Update allowselfservicepurchase-powershell.md (#7253)
MicrosoftDocs/microsoft-365-docs: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md

To use the **MSCommerce** PowerShell module, you need:

> - A Windows 10 or later Operating system.

Reviewer: ```suggestion
- A Windows 10 or later operating system.
```

- PowerShell 5 or below. Currently, PowerShell 6.x/7.x isn't supported with this module.

> - Global or Billing admin role for your tenant to change the MSCommerce Product Policies.

Reviewer 2: Change to:
The Global or Billing admin role for your tenant to change the **MSCommerce** product policies.

> - Global or Billing Admin role for your tenant to to change the Commerce Product Policies.

Reviewer: ```suggestion
- Global or Billing admin role for your tenant to change the MSCommerce Product Policies.
```

> - Global reader to only read Commerce Product Policies.

Reviewer: ```suggestion
- Global reader to only read MSCommerce Product Policies.
```

> - Global reader to only read MSCommerce Product Policies.

Reviewer 2: Change to:
The Global reader role for your tenant to see a read-only list of **MSCommerce** product policies.

To use the **MSCommerce** PowerShell module, you need:

- A Windows 10 or later operating system.
- PowerShell 5 or below. Currently, PowerShell 6.x/7.x isn't supported with this module.

> - Global or Billing admin role for your tenant to change the MSCommerce Product Policies.

Reviewer: ```suggestion
- The Global reader role for your tenant to see a read-only list of **MSCommerce** product policies.
```

> - Global or Billing admin role for your tenant to change the MSCommerce Product Policies.

Reviewer: ```suggestion
- The Global or Billing admin role for your tenant to change the **MSCommerce** product policies.
```

> - Global reader role for your tenant to see a read-only list of MSCommerce product policies.

Reviewer: ```suggestion
- The Global reader role for your tenant to see a read-only list of **MSCommerce** product policies.
```
